.github/workflows/depup.yml

```diff
@@ -21,7 +21,7 @@ jobs:
           repo: reviewdog/reviewdog
 
       - name: Create Pull Request
-        uses: peter-evans/create-pull-request@v6
+        uses: peter-evans/create-pull-request@v7
```

Comment:
No impact
https://github.com/peter-evans/create-pull-request/blob/main/docs/updating.md#updating-from-v6-to-v7